docs/configuration.md

```diff
@@ -128,10 +128,10 @@ ADMIN_PASSWORD needs to be set.
 ## APPLICATION_ROOT
 
 If empty, ihatemoney will be served at domain root (e.g:
```

Comment:
Perhaps this should be changed to "by default" since the default value is "/"

```suggestion
By default, ihatemoney will be served at domain root (e.g:
```